code/game/machinery/computer/crew.dm

```diff
@@ -33,9 +33,17 @@
 		src.updateDialog()
 		return
 
+var/datum/singleton/crewmonitor/crewmonitor = new

```

Comment:
I know what a singelton is, and it means nothing in byond's type paths.
a global or static is byond's way of telling a developer "there is only one of these"